src/app/data/forms/combobox.ts

```diff
@@ -6,6 +6,7 @@ import { FormControl } from './form-control';
 export class Combobox extends FormControl {
   options: Option[] = [];
   editable = false;
+  value = undefined;
```

Comment:
Da alle FormControls ein value haben, wäre es am besten, das in die abstrakte Basisklasse aufzunehmen, anstatt in jedem einzelnen FormControl